debian_radius/bionic/privacyidea-radius.install

```diff
@@ -1,4 +1,4 @@
-privacyidea_radius.pm	/usr/share/privacyidea/freeradius
+privacyidea_radius.pm	/etc/freeradius/3.0/mods-config/privacyidea-perl/
```

Comment:
Imho this is not the right location for the perl module, for the program code.
The mods-config file contains the configuration of modules, no the program code itself.

Reply:
fixed in a02820c